src/tests/unit/tests/injected/components/details-dialog.test.tsx

```diff
@@ -191,17 +217,11 @@ describe('DetailsDialog', () => {
                     dialogHandler: dialogDetailsHandlerMockObject,
                 };
 
-                const wrapper = shallow<DetailsDialog>(<DetailsDialog {...props} />);
+                render(<DetailsDialog {...props} />);
 
-                const commandBar = wrapper.find(CommandBar);
-                const onClick = commandBar.prop('onClickInspectButton');
+                getMockComponentClassPropsForCall(CommandBar).onClickInspectButton(eventStub);
 
-                onClick(eventStub);
-
-                clickHandlerMock.verify(
-                    handler => handler(wrapper.instance(), eventStub),
-                    Times.once(),
-                );
+                clickHandlerMock.verify(handler => handler(It.isAny(), eventStub), Times.once());
```

Comment:
See above comment for how to adapt this test to still check that the component is passed into the click handler.